### DISCUSSION TITLE
possible to have different hex sizes in the grid or similar

### DISCUSSION THREAD

**Discussion by s3638844** [flauwekeul/honeycomb#96] [Q&A]
wondering if it's possible to use this to strucuture a dashboard with 2 different hex sizes.

Or if a cluser of hex's could be made into a single in some fashion

**Comment by flauwekeul** [comment]
In short: during render time you could try to "balloon" a hex by multiplying its dimensions by a fixed value and render that (and maybe not render the hexes the bigger hex is now covering).

You could also try to fix it by having two grids: one with small hexes and one with bigger hexes and then rendering the latter on top of the former.

Let me know if you need more help!
